syntax = "proto3";

package greet;

service Greeter {
  rpc SayHello (HelloRequest) returns (HelloReply) {}
}

message HelloRequest {
  string name = 1;
  float cpu_usage = 2;
  int64 memory_usage = 3;
  int64 disk_usage = 4;
}

message HelloReply {
    string message = 1;
}